XLS-0094-dynamic-MPT/README.md

```diff
@@ -60,17 +60,19 @@ Bits in `MutableFlags` indicate specific fields or flags may be modified after i
-| `tfMPTCanMutateCanLock`     |  ️`0x0002`   |       2       | Indicates flag `lsfMPTCanLock` can be changed                                                            |
-| `tfMPTCanMutateRequireAuth` |  ️`0x0004`   |       4       | Indicates flag `lsfMPTRequireAuth` can be changed                                                        |
-| `tfMPTCanMutateCanEscrow`   |   `0x0008`   |       8       | Indicates flag `lsfMPTCanEscrow` can be changed                                                          |
-| `tfMPTCanMutateCanTrade`    |   `0x0010`   |      16       | Indicates flag `lsfMPTCanTrade` can be changed                                                           |
-| `tfMPTCanMutateCanTransfer` |  ️`0x0020`   |      32       | Indicates flag `lsfMPTCanTransfer` can be changed                                                        |
-| `tfMPTCanMutateCanClawback` |  ️`0x0040`   |      64       | Indicates flag `lsfMPTCanClawback` can be changed                                                        |
-| `tfMPTCanMutateMetadata`    | `0x00010000` |     65536     | Allows field `MPTokenMetadata` to be modified                                                            |
-| `tfMPTCanMutateTransferFee` | `0x00020000` |    131072     | Allows field `TransferFee` to be modified                                                                |
+`MutableFlags` are prefixed with `tmf` to clearly distinguish them from standard `tf` prefix used for `Flags`.
+
+| Flag Name                    |  Hex Value   | Decimal Value | Description                                                                              |
+| ---------------------------- | :----------: | :-----------: | ---------------------------------------------------------------------------------------- |
+| [Reserved]                   |  ️`0x0001`   |       1       | [Reserved; To align with `Flags` values, the `MutableFlags` value starts from `0x0002`.] |
+| `tmfMPTCanMutateCanLock`     |  ️`0x0002`   |       2       | Indicates flag `lsfMPTCanLock` can be changed                                            |
+| `tmfMPTCanMutateRequireAuth` |  ️`0x0004`   |       4       | Indicates flag `lsfMPTRequireAuth` can be changed                                        |
+| `tmfMPTCanMutateCanEscrow`   |   `0x0008`   |       8       | Indicates flag `lsfMPTCanEscrow` can be changed                                          |
+| `tmfMPTCanMutateCanTrade`    |   `0x0010`   |      16       | Indicates flag `lsfMPTCanTrade` can be changed                                           |
+| `tmfMPTCanMutateCanTransfer` |  ️`0x0020`   |      32       | Indicates flag `lsfMPTCanTransfer` can be changed                                        |
+| `tmfMPTCanMutateCanClawback` |  ️`0x0040`   |      64       | Indicates flag `lsfMPTCanClawback` can be changed                                        |
```

Comment:
For clarity, the hex values should use the full bit size, e.g. `0x00000001` instead of `0x0001`.